sinopia_client/src/api/LDPApi.js

```diff
@@ -54,6 +54,7 @@
      * @param {String} slug The suggested URI path for the group.
      * @param {module:model/LDPContainer} group Group metadata to insert into base container and describe the group.
      * @param {Object} opts Optional parameters
+     * @param {String} opts.link specifies container type.  you probably shouldn&#39;t override this parameter for this operation. (default to &lt;http://www.w3.org/ns/ldp#BasicContainer&gt;; rel&#x3D;&quot;type&quot;)
```

Comment:
Seeing more HTML entities in the markdown markup here. Could be sprinkled throughout.